src/maingoblint.ml

```diff
@@ -459,7 +459,7 @@ let main =
         | Exit ->
           exit 1
         | Timeout ->
-          Printexc.print_backtrace BatInnerIO.stderr;
+          (* Printexc.print_backtrace BatInnerIO.stderr; *)
```

Comment:
The reason I added this line when updating the timeout mechanism is because it used to print backtrace also before #172. I thought it can be useful for understanding timeouts because it at least tells which step (initializing, solving, verifying, witness, etc) the timeout happens, which _may_ be relevant.

But I also see how the entire backtrace can be annoying because of solver recursion making it uselessly deep.

Reply:
Yes. I think by default this should be off since it's not an exception but expected (the timeout).
Maybe we get the output clean enough such that the step it was in is easily visible/searchable.
Maybe it's a good idea to have a signal for `Printexc.print_backtrace` to see what it's currently doing. Seems like it can be called without a raised exception.

Reply:
I added `print_backtrace` on SIGUSR1 in https://github.com/goblint/analyzer/pull/207/commits/bb747e3115d6cfbe847db24fdc3210812c3226d2.
However, it only shows a trace until the re-raise here: https://github.com/goblint/analyzer/blob/bb747e3115d6cfbe847db24fdc3210812c3226d2/src/maingoblint.ml#L312-L313

Reply:
Ok, with `Printexc.get_callstack 999` it shows more. https://github.com/goblint/analyzer/pull/207/commits/cd6184349049ac274eb1a5deae5133f724f32796